.github/CODEOWNERS

```diff
@@ -522,6 +522,7 @@ x-pack/plugins/integration_assistant @elastic/security-scalability
 src/plugins/interactive_setup @elastic/kibana-security
 test/interactive_setup_api_integration/plugins/test_endpoints @elastic/kibana-security
 packages/kbn-interpreter @elastic/kibana-visualizations
+x-pack/plugins/observability_solution/inventory @elastic/obs-ai-assistant
```

Comment:
```suggestion
x-pack/plugins/observability_solution/inventory @elastic/obs-ux-infra_services
```

Reply:
cheers, will update this in the kibana.jsonc file